packages/module/patternfly-docs/content/examples/CatalogItemHeader.md

```diff
@@ -38,7 +38,7 @@ import pfLogo2 from './examples/pfLogo2.svg';
   title="Patternfly-React"
   vendor={
     <span>
-      provided by <a href="http://redhat.com">Red Hat</a>
+      provided by <a style={{ textDecoration: 'underline' }}href="http://redhat.com">Red Hat</a>
```

Comment:
This link started getting flagged as an a11y issue because of a lack of non-color link distinction. I'm not sure if some upstream changes may be needed.